.travis.yml

```diff
@@ -21,6 +21,15 @@ jobs:
     script:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest
+  - name: Minikube - Frontend
+    cache:
+      directories:
+        - ~/.npm
+        - ~/.cache
```

Comment:
I think the cache is unnecessary, since it only helps if you are building in stages.

Reply:
The cache setting is recommend by official cypress-travis document https://docs.cypress.io/guides/guides/continuous-integration.html#Travis
It is to cache both npm modules and Cypress binary

 NOTE:  Cypress is downloaded to `~/.cache/Cypress` in linux
https://docs.cypress.io/guides/getting-started/installing-cypress.html#Overriding-the-Binary-Cache-Folder